docs/bastion.md

```diff
@@ -68,6 +68,29 @@ spec:
       bastionPublicName: bastion.example.kubernetes.com
 ```
 
+### Using SSH agent to access your bastion
+
+Verify your local agent is configured correctly
+
+```
+$ ssh-add -L
+ssh-rsa <PUBLIC_RSA_HASH> /Users/kris/.ssh/id_rsa
+```
+
+If that command returns no results, add the agent to `ssh-agent`
```

Comment:
(Pre-existing typo): should be "add the key to `ssh-agent`" or something like that I think